apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -74,8 +79,10 @@ <h5>
                             {{localeData?.normal_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.normal?.symbol}}{{receiptsSummaryData?.normal?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.normal?.symbol}}{{receiptsSummaryData?.normal?.amount |
+                                    giddhCurrency}}/- -->
+                                    <amount-field [amount]="receiptsSummaryData?.normal?.amount" [currencySymbol]="receiptsSummaryData?.normal?.symbol" [currencyCode]="false">
```

Comment:
Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

Reply:
done